.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
```

Comment:
Do we still support node 12?

Reply:
Yes, it is not end-of-life until April 30th 2022.